perf-baseline/reporter/main.go

```diff
@@ -55,6 +57,8 @@ var (
 func main() {
 	namespace = flag.String("namespace", "l5d-perf", "namespace where the performance test is running")
 	logLevel := flag.String("log-level", log.InfoLevel.String(), "log level, must be one of: panic, fatal, error, warn, info, debug")
+	warmupTime = flag.Uint("warmup-time", 7*60, "Number of seconds to wait before starting to report numbers")
+	reportInterval = flag.Uint("report-interval", 10, "Interval in seconds at which statistics are dumped")
```

Comment:
Related to @adleong's comments, I think these flags would be more user-friendly as durations, and they'd be easier to work with in the code, which is already expecting durations. I'd change these too:

```
	warmupTime := flag.Duration("warmup-time", 7*time.Minute, "Time to wait before starting to report numbers")
	reportInterval := flag.Duration("report-interval", 1*time.Minute, "Interval at which statistics are dumped")
```

Then you could substitute those values directly into the `time.After` and `time.Tick` calls below.

Also note that since you're only using the values of these flags inside the `main` func, you don't need to define them as global vars on lines 38 and 39. I'd remove those two lines, and then use the `:=` assignment operator here, as show in the code snippet above.

Reply:
Oh, this is neat ! I come from the Python & Java world. Those are my first lines of Go code, so I was not expecting a command line option to be automatically parsed as a duration :) fancy